### ISSUE TITLE
Email: Permit images are missing alt text.

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#616]
## Notes
The 2 large permit images have no alt attribute value. 
Supply alt text that describes these 2 images. 

![image.png](https://images.zenhubusercontent.com/5913513251aa2c30577de73e/18296323-f9d2-4ecf-b2e3-f64f3142405b)

![image.png](https://images.zenhubusercontent.com/5913513251aa2c30577de73e/e82c0547-e19f-4624-a0d3-17e991c04b89)

Linkage: CTF-99
WCAG: 1.1.1 Non-text Content 

## Acceptance criteria
- [x] permit svg has alt text

## Tasks
- [ ] create text version of permit with placeholders for dynamic variables like stump height etc.
- [x] figure out how to attach to the email as alt text for the png and svg

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by sethalt** [comment]
@MelissaBraxton alt text for the permit image in the email. Please review and approve. 
<img width="1061" alt="screen shot 2018-03-26 at 4 08 10 pm" src="https://user-images.githubusercontent.com/2347759/37933154-04f00c6a-3110-11e8-81f7-15b1540657a4.png">

**Comment by sethalt** [comment]
After a conversation with @JohnHewittCriterion, we only need to describe in the alt text the information that is not already available to the use in the email. Including the text "Permit must be printed and filled out" suggests to an non-sighted user that they will need to ask for help to fill out the permit.

**Comment by MelissaBraxton** [comment]
This is a good start @sethalt!

The info re: permit placement on the dashoboard is in the body of the email, so maybe we don't need that in the alt text 

There is some info on the permit that is not in the email, in addition to what you have there: 

- It is unlawful to reuse or duplicate this permit for the purpose of removing more products than the permit holder has purchased. 
- Complete permit before transporting tree
- Have permit holder present at time of tree cutting

**Comment by sethalt** [comment]
Good catch @MelissaBraxton, i've updated with the following:
```Christmas tree permit 2018 for Mt. Hood National Forest. Permit must be printed and filled out with bold black ink on the date that you cut your tree. It is unlawful to reuse or duplicate this permit for the purpose of removing more products than the permit holder has purchased. Complete permit before transporting tree. Have permit holder present at time of tree cutting.```

**Comment by sadlerw** [comment]
deployed on staging - just needs approvals @MelissaBraxton @tram @ASprinkle

**Comment by MelissaBraxton** [comment]
Thanks @sethalt and @sadlerw  - What you have here looks good, but I'm not seeing it on staging.

**Comment by sethalt** [comment]
@MelissaBraxton this is on staging now.
